setup.py

```diff
@@ -53,11 +53,83 @@ def get_data_files():
+        'share/jupyterhub/static/components/jquery/external/sizzle/dist/sizzle.min.js',
+        'share/jupyterhub/static/components/jquery/external/sizzle/dist/sizzle.min.map',
+        'share/jupyterhub/static/components/jquery/dist/core.js',
+        'share/jupyterhub/static/components/jquery/dist/jquery.slim.min.js',
+        'share/jupyterhub/static/components/jquery/dist/jquery.slim.min.map',
+        'share/jupyterhub/static/components/jquery/dist/jquery.min.js',
+        'share/jupyterhub/static/components/jquery/dist/jquery.min.map',
+        'share/jupyterhub/static/components/moment/moment.js',
+        'share/jupyterhub/static/components/moment/LICENSE',
+        'share/jupyterhub/static/components/moment/ender.js',
+        'share/jupyterhub/static/components/moment/min/moment.min.js',
+        'share/jupyterhub/static/components/moment/min/locales.min.js',
+        'share/jupyterhub/static/components/moment/min/moment-with-locales.min.js',
+        'share/jupyterhub/static/components/font-awesome/css/font-awesome.css.map',
+        'share/jupyterhub/static/components/font-awesome/css/font-awesome.css',
+        'share/jupyterhub/static/components/font-awesome/css/font-awesome.min.css',
+        'share/jupyterhub/static/components/font-awesome/fonts/fontawesome-webfont.ttf',
+        'share/jupyterhub/static/components/font-awesome/fonts/FontAwesome.otf',
+        'share/jupyterhub/static/components/font-awesome/fonts/fontawesome-webfont.eot',
+        'share/jupyterhub/static/components/font-awesome/fonts/fontawesome-webfont.woff',
+        'share/jupyterhub/static/components/font-awesome/fonts/fontawesome-webfont.svg',
+        'share/jupyterhub/static/components/font-awesome/fonts/fontawesome-webfont.woff2',
+        'share/jupyterhub/static/components/requirejs/require.js',
+        'share/jupyterhub/static/components/requirejs/bin/r.js',
+        'share/jupyterhub/static/js/utils.js',
+        'share/jupyterhub/static/js/token.js',
+        'share/jupyterhub/static/js/jhapi.js',
+        'share/jupyterhub/static/js/home.js',
+        'share/jupyterhub/static/js/admin.js',
+        'share/jupyterhub/static/css/style.min.css',
+        'share/jupyterhub/static/css/style.min.css.map',
+        'share/jupyterhub/static/images/jupyterhub-80.png',
+        'share/jupyterhub/static/images/jupyter.png',
+        'share/jupyterhub/templates/stop_pending.html',
+        'share/jupyterhub/templates/404.html',
+        'share/jupyterhub/templates/page.html',
+        'share/jupyterhub/templates/login.html',
+        'share/jupyterhub/templates/spawn.html',
+        'share/jupyterhub/templates/admin.html',
+        'share/jupyterhub/templates/spawn_pending.html',
+        'share/jupyterhub/templates/error.html',
+        'share/jupyterhub/templates/home.html',
+        'share/jupyterhub/templates/oauth.html',
+        'share/jupyterhub/templates/logout.html',
+        'share/jupyterhub/templates/token.html',
```

Comment:
Rather than one file at a time, it's probably better to use directories, e.g. `.../templates/*.html`, `font-awesome/fonts/*`, etc. This will make it easier to avoid running into issues with missing files in the future.